seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -21,14 +22,38 @@
 class TerraApiUtilsCase(TestCase):
     fixtures = ['users', 'social_auth']
 
-    def test_is_google_authenticated(self):
-        user = User.objects.get(email = 'test_user@test.com')
+    @mock.patch('seqr.views.utils.terra_api_utils.SOCIAL_AUTH_GOOGLE_OAUTH2_KEY')
+    def test_is_google_authenticated(self, mock_social_auth_key):
+        user = User.objects.get(username='test_user')
         r = is_google_authenticated(user)
         self.assertTrue(r)
-        user = User.objects.get(email = 'test_local_user@test.com')
+
+        local_user = User.objects.get(username='test_local_user')
+        r = is_google_authenticated(local_user)
+        self.assertFalse(r)
+
+        mock_social_auth_key.__bool__.return_value = False
         r = is_google_authenticated(user)
         self.assertFalse(r)
 
+    @mock.patch('seqr.views.utils.terra_api_utils.anvil_enabled')
+    @mock.patch('seqr.views.utils.terra_api_utils.is_google_authenticated')
+    def test_is_anvil_authenticated(self, mock_is_google_authenticated, mock_anvil_enabled):
```

Comment:
Should it also add a test for a local logged in user like `test_is_google_authenticated`?

Reply:
no because `is_google_authenticated` is mocked and thats is where the local user check occurs. That case is tested explicitly in the test for that function, and that function is mocked and not tested here